src/general/CombinableActionData.cs

```diff
@@ -43,6 +48,21 @@ public virtual CombinableActionData Combine(CombinableActionData other)
         return CombineGuaranteed(other);
     }
 
+    /// <summary>
+    ///   Merge the other data into this if possible.
+    /// </summary>
+    /// <param name="other">The action to merge into this</param>
+    /// <returns>If a merge has been conducted</returns>
+    public virtual bool TryMerge(CombinableActionData other)
+    {
+        var mode = GetInterferenceModeWith(other);
+        if (mode != ActionInterferenceMode.Combinable && mode != ActionInterferenceMode.CancelsOut)
```

Comment:
I think this should instead check if the mode is combinable or not rather than this way because this logic will bite us if new action interference modes are ever added.

Reply:
If just combineable is checked, you get a new action every time you put the slider to the original place. Drag around original place slowly for several times and you'll see.

Reply:
It does cause some problems.

Reply:
I more meant like you should *reverse* this if check and not *only* check for combinable value. I did this in my refactoring (see my next comment).